com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -1199,11 +1068,11 @@ internal void HandleIncomingData(ulong clientId, Channel channel, ArraySegment<b
                                 else
                                 {
 #if DEVELOPMENT_BUILD || UNITY_EDITOR
-                                    s_MLAPIServerSTDRPCQueued.Begin();
+                                    s_ServerRpcQueued.Begin();
 #endif
-                                    InternalMessageHandler.RPCReceiveQueueItem(clientId, messageStream, receiveTime,RpcQueueContainer.QueueItemType.ServerRpc);
+                                    InternalMessageHandler.RpcReceiveQueueItem(clientId, messageStream, receiveTime,RpcQueueContainer.QueueItemType.ServerRpc);
 #if DEVELOPMENT_BUILD || UNITY_EDITOR
-                                    s_MLAPIServerSTDRPCQueued.End();
+                                    s_ServerRpcQueued.End();
```

Comment:
I see that there is some variable/function renaming in this PR unrelated to the original intent of the PR. I think we should generally try and keep those in a separate PR.

Reply:
I will revert these kind of naming changes back shortly.